configuration/pih/htmlforms/mentalHealth.xml

```diff
@@ -917,6 +917,31 @@
 
             </div>
         </div>
+
+           <div class="section-container" id="radio_button_container">
+
+            <div class="col">
+                <!-- <p>
+                    <label>
+                        comment
+                    </label>
+                    <obs conceptId="CIEL:163044" type="textarea"/>
+                </p> -->
```

Comment:
unless we really need to keep it for some reason, let's just remove this instead of commenting it out?

Reply:
what you ask me to remove is what we neek i think @mogoodrich